src/containers/Favorites/dispatcher.js

```diff
@@ -0,0 +1,13 @@
+import actions from './actions';
+import locationActions from '../Location/actions';
+
+const { loadFavorites, searchFavorites } = actions.creators;
+const { setPath } = locationActions.creators;
+
+const dispatcher = dispatch => ({
```

Comment:
This is good, you could also use `bindActionCreators` or the shorthand notation to avoid having to wrap all your creators with the `dispatcher`.